src/components/com_tjcertificate/site/views/certificates/view.html.php

```diff
@@ -133,6 +144,15 @@ public function display($tpl = null)
 		$this->manageOwn     = $this->user->authorise('certificate.external.manageown', 'com_tjcertificate');
 		$this->create	     = $this->user->authorise('certificate.external.create', 'com_tjcertificate');
 
+		if ($this->isAgencyEnabled)
+		{
+			$this->filterForm->removeField('user_id', 'filter');
+		}
+		else
+		{
+			$this->filterForm->removeField('agency_id', 'filter');
+		}
+
```

Comment:
try same code as suggested in the above files

Reply:
done